src/App.js

```diff
@@ -39,9 +43,14 @@ function App() {
     />
   ));
 
-  function addTask(name) {
+  function addTask(task, assignee) {
     // state could be used to check whether the task already exists
-    tasks.run((_state, enqueue) => enqueue(...taskAdded("todo-" + nanoid(), name)));
+    tasks.run((_state, enqueue) => enqueue(...taskAdded("todo-" + nanoid(), task, user, assignee)));
+  }
+
+  function addUser(name) {
+    setUser(name)
+    tasks.run((_state, enqueue) => enqueue(...userAdded(name)));
```

Comment:
This currently emits a UserAdded whenever the user signs in — which is fine, but perhaps unexpected. Another option would be to run this on the `users` fish and only emit the event if needed.

Reply:
Thanks & done.